31_HLSLPathTracer/app_resources/hlsl/render_rwmc_common.hlsl

```diff
@@ -0,0 +1,23 @@
+#ifndef _NBL_HLSL_PATHTRACER_RENDER_RWMC_COMMON_INCLUDED_
+#define _NBL_HLSL_PATHTRACER_RENDER_RWMC_COMMON_INCLUDED_
+#include "nbl/builtin/hlsl/cpp_compat.hlsl"
+
+#ifndef __HLSL_VERSION
+#include "matrix4SIMD.h"
+#endif
+
+struct RenderRWMCPushConstants
+{
+#ifdef __HLSL_VERSION
+	float32_t4x4 invMVP;
+#else
+	nbl::core::matrix4SIMD invMVP;
+#endif
+	int sampleCount;
+	int depth;
+	float start;
+	float base;
+	float kappa;
```

Comment:
dont use the same push constants for resolve and rendering & splatting 

I really want you to pack up `start` and `base` into its own `rwmc::SplattingParameters` struct